crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -1188,7 +1188,7 @@ where
     /// This is a no-op if the sequencer endpoint is not configured.
     #[cfg(feature = "optimism")]
     pub async fn forward_to_sequencer(&self, tx: &Bytes) -> EthResult<()> {
-        if let Some(endpoint) = self.network().sequencer_endpoint() {
+        if let Some(endpoint) = &self.inner.sequencer_client.sequencer_endpoint {
```

Comment:
this should be a function of the client type

Reply:
Could you elaborate more on this ?  So this should be an impl of the sequencer_client ?

Reply:
yeah, basically the comment below, 
transform this into `Option<SequencerClient>` where `SequencerClient`'s fields are non optional